### ISSUE TITLE
Unclear that `npm run build` must be run before `npm run serve`.

### ISSUE THREAD

**Issue by bobbyearl** [skylineos/clsp-videojs-plugin#37]
While following through the `README`, I get an error when running `npm run serve`:

```
ERROR in ./src/js/videojs-mse-over-clsp.js
Module not found: Error: Can't resolve './conduit/clspConduit.generated.js' in '/Users/bobby.earl/GitHub/clsp-videojs-plugin/src/js'
@ ./src/js/videojs-mse-over-clsp.js 4:0-44
@ multi ./src/js/videojs-mse-over-clsp.js
```

Running `npm run build` generates the missing file, but it wasn't clear from the docs that I needed to do this first.  Alternatively, I see I could run `npm run serve-watch` which does run `build` first.

**Comment by brainthinks** [comment]
@bobbyearl  - thanks for the contribution!  We recently updated the npm scripts and are at a new pre-release version, which addresses this issue.  Take a look at the updated readme and package.json, and let us know if you think the issue is resolved.

**Comment by brainthinks** [comment]
This issue has definitely been resolved with 0.15.0.
